test/binary/bad-opcode-prefix.txt

```diff
@@ -13,12 +12,5 @@ section(CODE) {
 }
 (;; STDERR ;;;
 000001a: error: unexpected opcode: 0xfe 0x7f
+000001a: error: unexpected opcode: 0xfe 0x7f
```

Comment:
why is this output duplicated? And, don't we want the same output as we had before, for any behavior that depends on the function contents?

Reply:
I changes the type of the test from `run-objdump-gen-wasm` to `run-gen-wasm-bad`.  The latter is designed specifically for this type of test where we are testing for a binary parse error.

The error occurs twice in all the `run-gen-wasm-bad` tests, they all of the duplication like this.  I think its because `run-gen-wasm-bad` runs two different tools under the hood: wasm-validate and wasm2wat, and checks that they both fail.

Reply:
ah ok, that makes sense.